### PR TITLE
Changed: Change the fetch module to isomorphic-fetch.

### PR DESCRIPTION
**Pull request by AtlasDev**
PR for issue #1 .
- Now using `isomorphic-fetch` instead of `node-fetch`. This means the module is compatible with browsers.
- Added documentation to the request function
- Fixed indentations in helpers.ts

### CODE REVIEW

File: src/lib/helpers.ts [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
-export async function request(uri: string, headers: { [header: string]: string; } = {}) {
-    const req = await fetch(uri, { compress: true, timeout: 30000, headers });
-    if (req.status !== 200) {
-        return null;
-    }
-    return await req.json();
+/**
+ * Make a http(s) request to a json api.
+ *
+ * Throws an error when a network error occured.
```
Comment (UnwrittenFun) [inline src/lib/helpers.ts]: No it doesn't..

Comment (AtlasDev) [inline src/lib/helpers.ts]: According to the [github documentation](https://github.github.io/fetch/#error), where the module is based upon, it does.

Comment (UnwrittenFun) [inline src/lib/helpers.ts]: fair enough, however i think some clarity about how that doesn't include 4xx and 5xx responses would be nice

**Comment by artdude543** [comment]
Can the return on the func args be removed please. This project does not use a prettier or formatter yet.

File: src/lib/helpers.ts [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+ *
+ * Rejects when a network error occured.
+ * 4xx and 5xx response codes are not network errors, and will resolve the promise.
+ *
+ * @param uri The uri to request the data from.
+ * @param headers Headers to attach to the request.
+ * @return The body of the response, parsed as json.
+ * It returns with `null` when the api does not respond with `200 OK`.
+ */
+export async function request(
```
Comment (artdude543) [inline src/lib/helpers.ts]: This does not need the extra return/formatting. Is this is preferred; then a separate PR should be made converting the entire code base to the same format.

**Comment by artdude543** [comment]
Did you also run the tests with the changes to make sure they all pass? I know you would need to set the config file up but it's explained in the default one on what you need and where to find it.

**Pull request merged**